bin/run_tests

```diff
@@ -187,18 +183,14 @@ test_poet_core(){
     run_docker_test ./consensus/poet/core/tests/test_consensus/unit_consensus.yaml -s poet
 }
 
-test_poet_families(){
-    run_docker_test ./consensus/poet/families/tests/test_validator_registry/tp_validator_registry.yaml \
-    -s validator
-}
-
 test_poet_simulator(){
     run_docker_test ./consensus/poet/simulator/tests/test_simulator/unit_simulator.yaml -s poet
 }
 
 test_python_sdk() {
     # transaction processors
     run_docker_test tp-config -s validator
+    run_docker_test tp-validator-registry -s validator
```

Comment:
The purpose of being able to pick modules is so you can save yourself time locally by only testing things that are affected by what you're working on. So the questions are "If I'm working on the validator registry, do I need to be testing the rest of the SDK?" and "If I'm working on the SDK, do I need to be testing the rest all of the transaction processors?" In any case, all the tests will be run on Jenkins, so it's really just a time saver.